docs/sources/calendar-schedules/web-schedule/calendar-export/index.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Export on-call schedules
+aliases:
+  - /docs/oncall/latest/calendar-schedules/web-schedule/calendar-export/
+canonical: https://grafana.com/docs/oncall/latest/calendar-schedules/web-schedule/calendar-export/
+description: "Learn how to export an on-call schedule from Grafana OnCall"
+keywords:
+  - Grafana
+  - oncall
+  - on-call
+  - calendar
+  - iCal export 
+weight: 500 
+---
+
+# Export on-call schedules
+
+Export on-call schedules from Grafana OnCall to your preferred calendar app with a one-time secret iCal URL. The schedule export allows you to add on-call schedules to your existing calendar to view on-call shifts alongside the rest of your schedule.  
+
+There are two schedule export options available:
+
+- **On-call schedule export** - Exports all on-call shifts for a particular schedule, including rotations, overrides, and assigned users.
+- **User-specific schedule export** - Exports assigned on-call shifts for a particular user. Use this export option to add your assigned on-call shifts to your calendar. 
+
```

Comment:
FYI (maybe we can add a note?), exports currently do not "resolve" the schedule, so you will get a calendar including all events (even those which are lower priority or overriden)